test/DataArrayTest.cpp

```diff
@@ -21,6 +22,38 @@ TEST_CASE("DataArrayCreation")
   REQUIRE(numTuples == 0);
 }
 
+TEST_CASE("complex::DataArray Copy TupleTest", "[complex][DataArray]")
+{
+  DataStructure dataStructure;
+  IDataStore::ShapeType tupleShape{5};
+  IDataStore::ShapeType componentShape{3};
+  Result<> result = CreateArray<int32>(dataStructure, tupleShape, componentShape, DataPath({"DataArray"}), IDataAction::Mode::Execute);
```

Comment:
Should this result be checked? Or would `DataArray<int32>::Create` be better here?

Reply:
Done